recipes/ozo/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import check_min_cppstd, Version
+import os
+import glob
+
+
+class OzoConan(ConanFile):
+    name = "ozo"
+    description = "C++ header-only library for asynchronous access to PostgreSQL databases using ASIO"
+    topics = ("ozo", "yandex", "postgres", "postgresql", "cpp17", "database", "db", "asio")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/yandex/ozo"
+    license = "PostgreSQL"
+
+    settings = "os", "arch", "compiler", "build_type"
+    requires = ("boost/1.75.0", "resource_pool/cci.20210322", "libpq/13.2")
+    generators = "cmake_find_package"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "7",
+            "Visual Studio": "15",
+            "clang": "5",
+            "apple-clang": "10",
+        }
+
+    def _validate_compiler_settings(self):
+        compiler = self.settings.compiler
+        if compiler.get_safe("cppstd"):
+            check_min_cppstd(self, "17")
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+
+        if not minimum_version:
+            self.output.warn("ozo requires C++17. Your compiler is unknown. Assuming it supports C++17.")
+        elif tools.Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration("ozo requires a compiler that supports at least C++17")
+
+    def validate(self):
+        if self.settings.os == "Windows":
+            raise ConanInvalidConfiguration("ozo does currently not support windows")
+
+        self._validate_compiler_settings()
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
```

Comment:
On mobile, but we have a new feature "strip_root" for the `tools.get` that can eliminate the glob + rename. Could you please apply that here?

Should be easy to find an example but I can give a better suggestion from my desktop (I can not multi line comment)

Reply:
Will do tomorrow, I carried most of this over from `resource_pool`.